templates/StatefulSet-zabbix-proxy.yaml

```diff
@@ -25,6 +25,13 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}-zabbix-proxy
         app.kubernetes.io/managed-by: {{ .Release.Service }}-zabbix-proxy
     spec:
+      nodeSelector:
+{{- toYaml .Values.nodeSelector | nindent 8 }}
+      tolerations:
+{{- toYaml .Values.tolerations | nindent 8 }}
+      affinity:
+{{- toYaml .Values.affinity | nindent 8 }}
+
```

Comment:
Hello @adamcharnock!

Thanks for your contribution!

I suggest the change for new lines because are more flexible. Do you agree?